### DISCUSSION TITLE
Εγγραφή για τις δωρεάν δυνατότητες φοιτητή

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#4] [Announcements]
Το Github παρέχει επιπλέον δωρεάν δυνατότητες αν πρώτα κάνετε ταυτοποίηση ως ενεργοί φοιτητές:

1. Αρχικά δημιουργείστε λογαριασμό στο [github](http://github.com) χρησιμοποιώντας το e-mail που σας έχει δοθεί από το ΕΑΠ. Σε περίπτωση που έχετε ήδη λογαριασμό με άλλο e-mail προσθέστε και το ΕΑΠ e-mail σας στο αντίστοιχο πεδίο του λογαριασμού σας.
2. Αποκτήστε δωρεάν πρόσβαση στο [GitHub Student Developer Pack](https://education.github.com/pack/join).

Οι δυνατότητες αυτές δεν είναι απαραίτητες για τις δραστηριότητες της ομάδας αλλά μπορούν να φανούν χρήσιμες στην εργασία σας ή στο μέλλον.

**Comment by ghost** [comment]
Χρειάζονται κάποιο έγγραφο στο οποίο να φαίνεται μέχρι πότε ισχύει η ιδιότητα μας (π.χ. φοιτητική ταυτότητα με πεδίο valid until). Δυστυχώς όμως οι ταυτότητες του ΕΑΠ δεν έχουν τη σχετική ένδειξη, γεγονός που μας δημιουργεί πρόβλημα και σε άλλες χρήσεις της ταυτότητας.

**Reply by epidrome** [reply]
Μήπως υπάρχει η δυνατότητα για κάποια πρόσφατη βεβαίωση φοίτησης από το μητρώο;

Πάντως οι επιπλέον δυνατότητες δεν μας είναι απαραίτητες
